testfiles/UnitVent5ZoneFixedOANoCoilOpt.idf

```diff
@@ -2,7 +2,7 @@
 !-Option OriginalOrderTop UseSpecialFormat
 !-NOTE: All comments with '!-' are ignored by the IDFEditor and are generated automatically.
 !-      Use '!' comments if they need to be retained when using the IDFEditor.
-! 5ZoneAirCooled.idf
```

Comment:
Obviously no problem with these fixups, but you can also just keep a tally of papercut issues like this and fix them in one non-functional-change PR so they don't interfere.  In this case, obviously it doesn't interfere, just noting it.